docs/src/get-started/add-data/merge.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Git merge
+weight: -9
+description: Add a service and see how data works between branches.
+---
+
+You have a separate environment with separate data.
+Next, add a service to your development environment.
+
+## Add a service
+
+Platform.sh includes many services such as databases, cache, and search engines.
+These are included in your project, so you can manage them with Git and back them up with your project.
+
+Add a database service (or choose [another service](../../add-services/_index.md)) by following these steps:
+
+1. Create a services configuration file.
+
+   ```bash
+   touch .platform/services.yaml
+   ```
+
+   This file holds the configuration for all services your app needs.
+
+2. Add a database in that file.
+   (If you need a different database service, you can choose from the [available services](../../add-services/_index.md#available-services).
+   Then change the `type` to fit your choice.)
+
+   ```yaml {location=".platform/services.yaml"}
+   db:
+       type: mariadb:10.5
+       disk: 1024
+   ```
+
+   Note that `db` is the name of the service.
+   You can give it any name you want with lowercase alphanumeric characters, hyphens, and underscores.
+
+3. Add a relationship between the database and your app in your app configuration:
+
+   ```yaml {location=".platform.app.yaml"}
+   relationships:
+       database: "db:mysql"
+   ```
+
+   This relationship is where connections are made.
+   The `database` is the name of the relationship, which you can change if you want.
+   The `db` has to be the same as the service name from step 2.
+
+4. Commit your changes and push:
+
+   ```bash
+   git add .
+   git commit -m "Add database and connect to app"
+   platform push
+   ```
+
+Now you have a database you can connect to your app.
+
+{{% get-started/connect-service %}}
+
+This example creates a table in your database, adds some data, prints the data as an HTML table,
+and deletes the database table.
+If you commit your changes and push, you should see the HTML table in your built app.
```

Comment:
🛠️  I think we can avoid the use of _should_ here? (Not always easy to avoid but I believe for results at least we can?) cf. https://developers.google.com/style/word-list#letter-s => An outcome is expected.

```suggestion
If you commit your changes and push, you see the HTML table in your built app.
```